designs/2021-init-command-eslint-cli/README.md

```diff
@@ -0,0 +1,139 @@
+- Repo: [eslint/eslint](https://github.com/eslint/eslint)
+- Start Date: 2020-06-29
+- RFC PR: https://github.com/eslint/rfcs/pull/79
+- Authors: Aniketh Saha ([anikethsaha](https://github.com/anikethsaha)) aladdin-add(weiran.zsd@outlook.com)
+
+# Move --init flag into a separate utility
+
+## Summary
+
+- Move the `init` command from main repo ([eslint](https://github.com/eslint/eslint)) to a new repo named `@eslint/create`.
```

Comment:
After thinking about this a bit more, I think eslint/create-config is probably a better name. That way we can do:

npm init @eslint/config

Which just looks more like it describes what is happening.

Reply:
awesome idea!